File: app/src/main/java/com/example/focusstartsecondpart/features/events/domain/model/Guest.java
package com.example.focusstartsecondpart.features.events.domain.model;

public class Guest {

    private String firstName;
    private String lastName;
    private String text;

    Guest(String firstName, String lastName, String text){
        this.firstName = firstName;
        this.lastName = lastName;
        this.text = text;
    }

    public String getFirstName() {
        return firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public String getText() {
        return text;
    }
}

File: app/src/main/java/com/example/focusstartsecondpart/features/guests/domain/model/Guest.kt
package com.example.focusstartsecondpart.features.guests.domain.model

import androidx.room.Entity
import androidx.room.PrimaryKey


@Entity
class Guest(@field:PrimaryKey
            var id: Int, val firstName: String, val lastName: String, val phone: String, var visited: Boolean) {
    var eventId: Int = 0
}
